tests/unit/wmg_processing/test_data_artifact_correctness.py

```diff
@@ -32,29 +27,6 @@ def find_dim_option_values_pandas(criteria: Dict, snapshot: WmgSnapshot, dimensi
 
 
 class DataArtifactCorrectness(unittest.TestCase):
-    # creates the expression summary default cube by group-by and summing the expression summary cube
-    # and checks to makes sure that the result is the same as the existing fixture (expression_summary_default)
-    def test__expression_summary_default_cube_correctness(self):
```

Comment:
this test is no longer relevant. it asserts that taking the expression summary cube and group-by/summing only the default logical dimensions should result in the same thing as the expression summary default cube. however, since we're adding the sqsum array to only the default cube and not the expression summary cube, this is no longer expected to be true.